### ISSUE TITLE
[Adaptive Cards- Import- Activity Update]: Talkback does not announce any information after activating 'Next' and 'Previous' month button under DatePicker.

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7260]
https://microsoft.visualstudio.com/OS/_workitems/edit/34086636
